setup.py

```diff
@@ -48,9 +48,9 @@ def main():
         scripts=[],
         zip_safe=False,
         install_requires=[
-            'requests>=2.7.0',
-            'Pillow==6.2.2',
-            'six>=1.10.0'
+            'requests>=2.31.0',
```

Comment:
Bump to the latest version with no vulnerabilities per Snyk.io

https://security.snyk.io/package/pip/requests
https://security.snyk.io/package/pip/pillow
https://security.snyk.io/package/pip/six